README.md

```diff
@@ -56,3 +57,6 @@ Any time you want to sync your project with the defaults, update this dependency
 
 [cover]: https://codecov.io/gh/webpack-contrib/webpack-defaults/branch/master/graph/badge.svg
 [cover-url]: https://codecov.io/gh/webpack-contrib/webpack-defaults
+
+[size]: https://packagephobia.now.sh/badge?p=webpack-defaults
```

Comment:
please remove these changes from the main README. they won't add any value to webpack-defaults' README

Reply:
👍